.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Purpose
```

Comment:
Is the name of this file a special name that gets automatically picked up by GH?